libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -59,6 +59,8 @@ class AP_Motors {
         MOTOR_FRAME_TYPE_DJI_X = 13, // X frame, DJI ordering
         MOTOR_FRAME_TYPE_CW_X = 14, // X frame, clockwise ordering
         MOTOR_FRAME_TYPE_I = 15, // (sideways H) octo only
+        MOTOR_FRAME_TYPE_NYT_PLUS = 16, // plus frame, no differential torque for yaw
```

Comment:
Don't forget to add these to the FRAME_TYPE parameter description!  Perhaps with "(TailSitter only)" appended